CMakeLists.txt

```diff
@@ -48,8 +61,13 @@ set(PDC_PACKAGE_VERSION_MAJOR "${PDC_VERSION_MAJOR}.${PDC_VERSION_MINOR}")
 set(PDC_PACKAGE_VERSION_MINOR "${PDC_VERSION_PATCH}")
 set(PDC_PACKAGE_STRING "${PDC_PACKAGE_NAME} ${PDC_PACKAGE_VERSION}")
 set(PDC_PACKAGE_TARNAME "${PDC_PACKAGE}")
-if(NOT MPI_RUN_CMD)
-    set(MPI_RUN_CMD mpiexec)
+
+
+#------------------------------------------------------------------------------
+# Setup CMake Prefix Paths for searching external libraries.
+#------------------------------------------------------------------------------
+if(NOT CMAKE_PREFIX_PATH)
```

Comment:
Why are we setting this explicitly like this?

Reply:
Please refer to https://github.com/hpc-io/pdc/pull/97